com.pengaru.glimmer.json

```
@@ -0,0 +1,30 @@
+{
+    "app-id" : "com.pengaru.glimmer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
```

this runtime version has been EOL for quite some time.
```suggestion
    "runtime-version" : "42",
```